### PR TITLE
Adding CAMP L2

### PR DESCRIPTION
**Pull request by hydrogenbond007**
pr to add camp network testnet to chainlist, here is the [developer documentation](https://docs.campnetwork.xyz/building-on-camp/add-camp-network#how-to-connect-to-camp-network)

### CODE REVIEW

File: _data/chains/eip155-325000.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Camp Network Testnet V2",
+  "chain": "CampL2",
+  "icon": "CampL2",
```
Comment (macvts) [inline _data/chains/eip155-325000.json:4]: the submitted pr didn't have icon, so rather take this out or add up icon file

**Review by macvts — COMMENTED** [review]
make sure to sync forked repo to updated with reference from original branch

**Comment by hydrogenbond007** [comment]
@macvts removed logo and updated the branch

**Review by ligi — CHANGES_REQUESTED** [review]
add parent field

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `CAMP` more than once

**Comment by hydrogenbond007** [comment]
@ligi Sorry kinda confused, here did you found the name twice?

**Comment by hydrogenbond007** [comment]
Oh seems like you are referencing to #4878, that pr was for Camp Testnet V1 and has been depreciated, seems like #5331 was trying to accomplish the testnet v2 upgrade.

**Pull request closed without merging**